src/main/java/com/questhelper/QuestHelperConfig.java

```diff
@@ -38,14 +44,60 @@
 {
 	enum QuestOrdering
 	{
-		A_TO_Z,
-		OPTIMAL;
+		/** Sort quests in alphabetical order */
+		A_TO_Z(QuestOrders.sortAToZ()),
+		/** Sort quests in reverse alphabetical order */
+		Z_TO_A(QuestOrders.sortZToA()),
+		/** Sort quests according to the Optimal Quest Guide (https://oldschool.runescape.wiki/w/Optimal_quest_guide) */
+		OPTIMAL(QuestOrders.sortOptimalOrder()),
+		;
+
+		private final Comparator<QuestHelper> comparator;
+		QuestOrdering(Comparator<QuestHelper> comparator) {
+			this.comparator = comparator;
+		}
+
+		public List<QuestHelper> sort(Collection<QuestHelper> list) {
+			return list.stream().sorted(getComparator()).collect(Collectors.toList());
```

Comment:
You can call `sort` on collections directly, however that modifies that collection. This way allows us to keep that collection intact and get a new list in return.